docs/USER.md

```diff
@@ -270,6 +271,26 @@ Please note that SPI service didn't perform any specific validity checks
 for the user provided credentials, so it's assumed user provides a correct data.
 
 
+## Overriding default binding lifetime
+In general, bindings (and their dependent secrets) are not supposed to be long-lived. Their TTL is configured globally, 
+and by defaults to 2 hrs. Some tasks may require the credentials to be available for a longer time, so the creator of the binding
```

Comment:
... and defaults to 2 hrs.
or
... and is 2 hrs by default.

Reply:
fixed